t04_refactored.py

```diff
@@ -211,9 +211,41 @@ def team_13_adv(username, delay):
 
 
 def team_14_adv(username, delay):
-    pass
-    # TODO Add your code here
-    return True
+    '''https://docs.google.com/document/d/1scXuIBzrGZxe0pZ-vEulOikC7Vc0QR5suwVhLVz6M9o/edit
+        Chris Georgiev
+        Dylan Roland
+        Kamau Clark'''
+
+    isDead = False
+
+    direction = input("Hey " + username + ", do you want to go North(1) or South(2)?: ")
+
+    if direction == "1":
+        print("You didn't die. In fact, you made it out of the cave and back into society. Yay.")
+        sleep(delay)
+
+    elif direction == "2":
+        print("You unfortunately fell off a cliff and are about to die.")
+        isDead = True
```

Comment:
Replace this with `return False` to kill your user. See the main function for how we kill our user now.